### ISSUE TITLE
'SearchFacebookPlaces' Test causing build failure on appveyor CI

### ISSUE THREAD

**Issue by theShiva** [InstaSharp/InstaSharp#67]
The `SearchFacebookPlaces()` test in causing the Instasharp solution build on Continuous Integration service appveyor.com to fail.

Here's the relevant build output.

``` msbuild
Passed   SearchFourSquareV2 
Failed   SearchFacebookPlaces 
Error Message: 
   Assert.AreEqual failed. Expected:<1.379678091>. Actual:<1.378994349>.  
Stack Trace: 
   at InstaSharp.Tests.Locations.<SearchFacebookPlaces>d__10.MoveNext() in c:\projects\instasharp\src\InstaSharp.Tests\Locations.cs:line 52
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
 
Passed   Get
```

The test is trying to locate some obscure facebook location called **Nanyang Polytechnic** located in Singapore.

``` C#
[TestMethod, TestCategory("Locations.Search")]
public async Task SearchFacebookPlaces()
{
    var result = await locations.Search(104999729569954);
    var locationItem = result.Data.Single();
    Assert.AreEqual(1.379678091M, locationItem.Latitude);
    Assert.AreEqual(103.848876663M, locationItem.Longitude);
    Assert.AreEqual(locationItem.Id, 906462);
    Assert.AreEqual("Nanyang Polytechnic", locationItem.Name);
}
```

**Comment by theShiva** [comment]
I tried to locate a more prominent, and permanent location like the [Facebook HQ in Menlo Park, CA](https://www.facebook.com/pages/Facebook-HQ/166793820034304).

`facebook_places_id =166793820034304`

When I ran this ID through the Instagram API endpoint on the console at apigee.com, I still get `latitude`, `longitude` values that don't match what google maps has for `latitude`, `longitude` for the same address and location.

[Click here](https://snap.apigee.com/1IARQC6) for the <kbd>Request</kbd>/<kbd>Response</kbd> Snapshot on apigee.com

![image](https://cloud.githubusercontent.com/assets/1057194/9153780/7bc97de0-3e23-11e5-8476-d3bd6c82b3c4.png)

On google maps, the same  [Facebook HQ in Menlo Park, CA](https://www.google.com/maps/place/Facebook/@37.4848474,-122.1483859,17z/data=!4m6!1m3!3m2!1s0x808fbc96cc9eae65:0xe4b691264d679dd6!2sFacebook!3m1!1s0x808fbc96cc9eae65:0xe4b691264d679dd6) shows `latitude`, `longitude` values that don't match exactly, after the 3rd or 4th decimal.

##### Instagram API Result for Facebook HQ Latitude, Longitude
    "latitude": 37.482898
    "longitude": -122.149909

##### Google Maps Result for Facebook HQ Latitude, Longitude
    "latitude": 37.4848474
    "longitude":-122.1483859

##### [Texas A&M Geoservices online](http://geoservices.tamu.edu/Services/Geocode/Interactive/) Result for Facebook HQ Latitude, Longitude 
    "latitude": 37.436934
    "longitude":-122.193602

In summary, none of the 3 results are exact matches for geo co-ordinates.

I tried this also for a location that will never move for the next 100 years i.e. the [Golden Gate Bridge in San Francisco](https://www.facebook.com/pages/Golden-Gate-Bridge/111856692159256). Same results, i.e. no two sources match exactly.

So I vote to modify this test, to only validate the `Location.ID` field, and instead of checking for the exact `latitude`, `longitude` values also, we can simply `Assert` that the returned values are not `0.0M` (the default for `decimal` data type) for the `latitude`, `longitude`.
